### PR TITLE
gh-1194: Add Windows command for patchcheck to Quick Reference

### PR DESCRIPTION
**Pull request by fipachu**
I think it will make life a little easier for first time contributors.

----
:books: Documentation preview :books:: https://cpython-devguide--1195.org.readthedocs.build/

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ instructions please see the :ref:`setup guide <setup>`.
    not require any issue to be created.
 
 6. Once you fixed the issue, run the tests, run ``make patchcheck``, and if
-   everything is ok, commit.
+   everything is ok, commit. On windows run ``.\python.bat Tools\patchcheck\patchcheck.py``
```
Comment (hugovk) [inline index.rst:61]: Perhaps something like:

```rst
6. Once you fixed the issue, run the tests,
   run ``make patchcheck`` (Unix/macOS)
   or ``.\python.bat Tools\patchcheck\patchcheck.py`` (Windows),
   and if everything is ok, commit.
```

I'm using macOS; someone on Windows will have to confirm if this command works.

Comment (fipachu) [reply index.rst:61]: Yeah, this is clearer. I'm probably gonna modify it tomorrow.

I got the command from [here](https://devguide.python.org/getting-started/pull-request-lifecycle/#patchcheck), but I'm guessing it's still necessary to have someone confirm it.

Comment (Mariatta) [reply index.rst:61]: We should also use Furo's tab feature to display the commands in macOS / windows.
https://pradyunsg.me/furo/reference/tabs/


e.g. 


```
.. tab:: macOS

    make patchcheck

.. tab:: Windows

    .\python.bat Tools\patchcheck\patchcheck.py 

```

Comment (ezio-melotti) [inline index.rst:61]:
```suggestion
   everything is ok, commit. On Windows run ``.\python.bat Tools\patchcheck\patchcheck.py``
```

**Pull request merged**